docs/content/TextInput.mdx

```diff
@@ -81,6 +81,12 @@ TextInput is a form component to add default styling to the native text input.
 <TextInput contrast aria-label="Zipcode" name="zipcode" placeholder="Find user" autoComplete="postal-code" />
 ```
 
+## Monospace text input
+
+```jsx live
+<TextInput monospace aria-label="personal access token" name="pat" placeholder="github_pat_abcdefg123456789" />
```

Comment:
after checking out the deployment preview I realised my earlier suggestion wasn't quite right, my bad.

I think you should switch `placeholder` to value so it looks more like product version. Would be nice to get the CheckIcon in there too!

```suggestion
<TextInput monospace leadingVisual={CheckIcon} aria-label="personal access token" name="pat" value="ghp_abcdefg123456789" />
```

Reply:
done 👍